docs/howtos/cluster-resource-override.md

```diff
@@ -0,0 +1,208 @@
+    - group: rbac.authorization.k8s.io
+      kind: ClusterRole
+      version: v1
+      name: secret-reader
+  policy:
+    overrideRules:
+      - clusterSelector:
+          clusterSelectorTerms:
+            - labelSelector:
+                matchLabels:
+                  env: prod
+        jsonPatchOverrides:
+          - op: remove
+            path: /rules/0/verbs/2
+          - op: remove
+            path: /rules/0/verbs/1
+```
+The `ClusterResourceOverride` object above will remove the verbs "list" and "watch" in the `ClusterRole` named 
+`secret-reader` on clusters with the label `env: prod`.
+
+## Applying the ClusterResourceOverride
+Create a ClusterResourcePlacement resource to specify the placement rules for distributing the cluster resource overrides across
+the cluster infrastructure. Ensure that you select the appropriate resource.
+```yaml
+apiVersion: placement.kubernetes-fleet.io/v1beta1
+kind: ClusterResourcePlacement
+metadata:
+  name: crp
+spec:
+  resourceSelectors:
+    - group: rbac.authorization.k8s.io
+      kind: ClusterRole
+      version: v1
+      name: secret-reader
+  policy:
+    placementType: PickAll
+    affinity:
+      clusterAffinity:
+        requiredDuringSchedulingIgnoredDuringExecution:
+          clusterSelectorTerms:
+            - labelSelector:
+                matchLabels:
+                  env: prod
+```
+The `ClusterResourcePlacement` configuration outlined above will disperse resources across all clusters labeled with `env: prod`. 
+As the changes are implemented, the corresponding `ClusterResourceOverride` configurations will be applied to the 
+designated clusters, triggered by the selection of matching cluster role resource `secret-reader`.
+
+To ensure that the `ClusterResourceOverride` object is applied to the selected resources, verify the `ClusterResourcePlacement`
+status:
+```
+Status:
+  Conditions:
+    ...
+    Last Transition Time:   2024-04-27T04:18:00Z
+    Message:                The selected resources are successfully overridden in the 10 clusters
+    Observed Generation:    1
+    Reason:                 OverriddenSucceeded
+    Status:                 True
+    Type:                   ClusterResourcePlacementOverridden
+    ...
+  Observed Resource Index:  0
+  Placement Statuses:
+    Applicable Cluster Resource Overrides:
+      example-cro-0
+    Cluster Name:  member-50
+    Conditions:
+      ...
+      Message:               Successfully applied the override rules on the resources
+      Observed Generation:   1
+      Reason:                OverriddenSucceeded
+      Status:                True
+      Type:                  Overridden
+     ...
+```
+The `ClusterResourcePlacementOverridden` condition indicates whether the resource override has been successfully applied
+to the selected resources in the selected clusters.
+
+Each cluster maintains its own `Applicable Cluster Resource Overrides` which contain the cluster resource override snapshot

```

Comment:
Example of `Applicable Cluster Resource Overrides`?

Reply:
I have re-arranged the order of the description, but the status above has an example of `Applicable Cluster Resource Overrides`.